lib/jekyll-admin/server/configuration.rb

```diff
@@ -2,12 +2,15 @@ module JekyllAdmin
   class Server < Sinatra::Base
     namespace "/configuration" do
       get do
-        json raw_configuration.to_liquid
+        json({
+          :content     => parsed_configuration,
+          :raw_content => raw_configuration,
+        })
```

Comment:
For the tests to pass, since you put the configuration under `content` & `raw_content` fields, you need to fix the  line [here](https://github.com/jekyll/jekyll-admin/blob/master/spec/jekyll-admin/server/configuration_spec.rb#L11) (and the following ones as well if necessary) accordingly.

```ruby
expect(last_response_parsed["content"]["foo"]).to eql("bar")
```